drivers/watchdog/wdt_nrfx_wdt.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2018, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrfx_wdt.h>
+#include <nrf_clock.h>
+#include <clock_control.h>
+#include <watchdog.h>
+
+#define SYS_LOG_DOMAIN "WDT NRF"
```

Comment:
"wdt_nrfx_wdt"
But shouldn't we name this file just "wdt_nrfx.c" (and then use "wdt_nrfx") here?